lisa/transformers/dom0_kernel_installer.py

```diff
@@ -35,6 +35,14 @@ class BinaryInstallerSchema(BaseInstallerSchema):
         ),
     )
 
+    # kernel config local absolute path
+    kernel_config_path: str = field(
+        default="",
+        metadata=field_metadata(
+            required=True,
```

Comment:
Does it mean this will be a required field ? If so, we should rather it optional

Reply:
@pupacha no, default would be empty string even if we dont pass it from runbook. 

If some value is set, then only we will try to copy the file to target system

Reply:
You can refer initrd parameter in this transformer, We are not passing it from runbook in our stack

Reply:
Ok. Thanks for confirmation.